example/docker-compose.yml

```diff
@@ -2,19 +2,22 @@
 version: '3.4'
 
 x-common-variables: &wikibase_variables
-  DB_SERVER: mysql.svc:3306
   MW_ADMIN_NAME: ${MW_ADMIN_NAME}
   MW_ADMIN_PASS: ${MW_ADMIN_PASS}
```

Comment:
Handing in all passwords as environment variables to all containers might be suboptimal security wise. Getting a shell in one container would reveal all the passwords of the stack.

Reply:
Good point, and agree on that. Will probably amend this, but have a larger change set that may wish to propose before iterating that... TBD.